dashboards_bundlers/local_deploy/static/main.js

```diff
@@ -38,39 +38,46 @@ requirejs.config({
 
 requirejs(['urth/dashboard'], function(Dashboard) {
     Dashboard.init().then(function() {
+        var IPython = window.IPython;
+        var widgetManager = IPython.notebook.kernel.widget_manager;
+        
         // Ugly, because we're special-casing declarative widget support, but
         // they need to be pre-loaded onto the page before we begin executing
         // any notebook code that depends on them.
-
-        // Declarative widgets 0.2.x subclass the WidgetModel base class
-        // to guarantee message sequence. They register this new base with
-        // the widget manager. Because the Thebe build includes these
-        // modules and initializes the manager singleton, we need to map
-        // them across from the Thebe loader to our loader to avoid making
-        // second, independent copies.
-        // All this goes away when we can use jupyter-js-services and
-        // ipywidgets independently.
-        define('nbextensions/widgets/widgets/js/manager', function() {
-            return {
-                // The kernel object has a reference to the widget manager
-                // (for now at least ...)
-                WidgetManager: IPython.notebook.kernel.widget_manager.constructor
+        define('jupyter-js-widgets', function() {
+            // All hail the monkey patch: override new_widget so that the 
+            // registry is used to fetch DeclWidgetModel instead of 
+            // almond-flavored requirejs from Thebe which we can't configure
+            var old_new_widget = IPython.notebook.kernel.widget_manager.new_widget;
+            widgetManager.new_widget = function(options) {
+                if(options.model_module && options.model_module.indexOf('jupyter-decl-widgets') === 0) {
+                    options.model_module = null;
+                }
+                // Invoke the original
+                return old_new_widget.apply(IPython.notebook.kernel.widget_manager, arguments);
             };
```

Comment:
Should we move the chunk above down with `register_widget_model`? Seems odd to have it here in the `define` for *jupyter-js-widgets*.